Makefile

```diff
@@ -389,4 +388,44 @@ deploy-docs: venv
 demo-update:
 	./hack/demo/generate-asciidemo.sh -u -n catalogd-demo catalogd-demo-script.sh
 
+# Catalogd targets
```

Comment:
Instead to add the targets here
We can just add the commands under the the current targets
also, not that we have the `CATALOG_IMAGE_REPO` in this Makefile already. 
So, we should not duplicate it.

Reply:
Thanks. This should be resolved.